system-addon/content-src/styles/_theme.scss

```diff
@@ -89,6 +89,7 @@ body {
   // Modal + overlay
   --newtab-modal-color: $grey-80;
   --newtab-overlay-color: $grey-90;
+  --newtab-overlay-color: rgba($grey-90, 0.8);
```

Comment:
Does the previous definition need to get removed?

Reply:
uhm. yes. indeed.